test/test_cp2k_mock.py

```diff
@@ -63,6 +65,20 @@ def mock_runner(mocker_instance, jobname: str) -> CP2K_Result:
     return run_mocked
 
 
+def test_deepcopy(mocker):
+    """Test the copy of a result instance."""
+    job = cp2k(templates.singlepoint, ETHYLENE)
+    jobname = "cp2k_job"
+    run_mocked = mock_runner(mocker, jobname)
+    result = run_mocked(job)
+    copy_result = copy.deepcopy(result)
+
+    init_restart(folder=WORKDIR)
+    assertion.eq(copy_result.molecule, result.molecule)
```

Comment:
In principle you could also use ``assertion.is_()`` here, as the deepcopy is in fact rather shallow.

Reply:
let's do that!